sharktank/build_tools/build_linux_package.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# build_linux_package.sh
+#
+# Builds sharktank Python package for Linux
+#
+# Usage:
+#   ./build_tools/build_linux_package.sh
+
+set -xeu -o errtrace
+
+THIS_DIR="$(cd $(dirname $0) && pwd)"
+REPO_ROOT="$(cd "$THIS_DIR"/../../ && pwd)"
+OUTPUT_DIR="${OUTPUT_DIR:-${THIS_DIR}/wheelhouse}"
+
+python -m pip wheel --disable-pip-version-check -v -w "${OUTPUT_DIR}" "${REPO_ROOT}/sharktank"
+
+wheel_output="$(echo "${OUTPUT_DIR}/sharktank-"*".whl")"
+ls "${wheel_output}"
+
+echo "Deleting other wheels"
+pushd ${OUTPUT_DIR}
+ls | grep -xv "sharktank-"*".whl" | xargs rm
+popd
```

Comment:
Any suggestions here? This style of deleting "everything but sharktank.whl" worries me for local use, if a bug is introduced... Maybe rework the .yml workflow to only upload a specific file, instead of the entire `OUTPUT_DIR`?

Reply:
Is there a way to tell pip to put the other wheels in a separate directory, or delete them afterwards maybe?

Logs with the list of what gets downloaded (2GB+ from torch, cuda, etc.): https://github.com/ScottTodd/SHARK-Platform/actions/runs/11621153296/job/32364368882#step:4:58

Reply:
Giving `--no-deps` a try... https://pip.pypa.io/en/stable/cli/pip_wheel/#cmdoption-no-deps

Reply:
That seems to work :D